package.json

```diff
@@ -107,6 +107,6 @@
     ]
   },
   "engines": {
-    "node": "^12.14.1 || ^14.15.0"
+    "node": "^12.13.0 || ^14.15.0 || >=15.0.0"
```

Comment:
Just out of curiosity - why lower the 12.x version here?

Reply:
copy from https://github.com/facebook/jest/blob/master/package.json#L147 . Angular requires minimum 12.14.1 starting from v12 while v9, v10, v11 doesn't require minimum 12.14.1, kind of hard to combine between Jest and Angular Node version support.